lint_test/avoid_using_api/named_parameter_ban/lib/named_parameter_ban_test.dart

```diff
@@ -0,0 +1,28 @@
+// ignore_for_file: unused_local_variable
+
+import 'package:external_source/external_source.dart';
+
+void testingBannedCodeLint() async {
+  // expect_lint: avoid_using_api
+  final bannedCodeUsage = BannedCodeUsage(parameter1: '');
+
+  // expect_lint: avoid_using_api
+  final test3 = BannedCodeUsage.test3(parameter1: '');
+
+  // expect_lint: avoid_using_api
+  final test5 = BannedCodeUsage.test5(parameter1: '');
```

Comment:
What is the difference between these 2 test cases? That looks like it is testing the same piece of functionality here.